inventree_digikey/Inventree.py

```diff
@@ -19,16 +21,51 @@
 PASSWORD = None
 API = None
 
+catagory_map = {}
+params_map = {}
+htsus_pk = None
+
 def load_config():
     config = configparser.ConfigParser()
     config.read(CONFIG_FILE_PATH)
     global API_URL, USERNAME, PASSWORD, API
+    global catagory_map, params_map, htsus_pk
 
     API_URL = config['INVENTREE_API']['URL']
     USERNAME = config['INVENTREE_API']['USER']
     PASSWORD = config['INVENTREE_API']['PASSWORD']
     API = InvenTreeAPI(API_URL, username=USERNAME, password=PASSWORD)
```

Comment:
I'll push this off into another PR but I like the idea of making the config file it's own module. Python global variables are kinda icky and I'm not a huge fan of how I did this. But having one module to read the ini file and map to variable locations could make extending the config.ini easier in the future

Reply:
I think the idea of the config modules is great.  I agree about globals == Bad.  Was just looking to minimize change at this time and followed your lead.  But looked over the config changes and that looks great.